cookbook/controller/argument_value_resolver.rst

```diff
@@ -0,0 +1,163 @@
+.. index::
+   single: Controller; Argument Value Resolvers
+
+Extending Action Argument Resolving
+===================================
+
+.. versionadded:: 3.1
+    The ``ArgumentResolver`` and value resolvers are added in Symfony 3.1.
+
+In the book, you've learned that you can add the :class:`Symfony\\Component\\HttpFoundation\\Request`
+as action argument and it will be injected into the method. This is done via the
```

Comment:
Maybe "In the book, you've learned that you can get the `Request` object by adding a `Request` argument to your controller."?